itest/assertions.go

```diff
@@ -722,13 +723,29 @@ func assertUniverseRootsEqual(t *testing.T, a, b *unirpc.AssetRootResponse) {
 	}
 }
 
+func succeedEventually(t *testing.T, f func(*testing.T),
+	timeout, retryInterval time.Duration) {
+
+	t.Helper()
+
+	require.Eventually(t, func() bool {
```

Comment:
Ahh, this super useful! 

Otherwise you need to modify things to return an error for the usual `wait` instances.